releasenotes/notes/python311-tracer-a7077c0e461622d2.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    tracer: added support for Python 3.11.
```

Comment:
add verbose prelude mentioning initial support for tracer, profiler and dynamic instrumentation is not yet compatible.